addons/website_quote/views/website_quote_templates.xml

```diff
@@ -18,7 +18,7 @@
                     <tr>
                         <th>Products</th>
                         <th>Quantity</th>
-                        <th>Taxes</th>
+                        <th><span t-if="any([line.tax_id for line in quotation.order_line])">Taxes</span></th>
```

Comment:
Shouldn't there be a place where we hide the `<div>`'s or `<td>`'s of the values of the taxes too ?